header/interface.go

```diff
@@ -60,12 +60,18 @@ type Exchange interface {
 	Getter
 }
 
+// RequestSize defines tha max amount of headers that can be requested/handled at once.
+var RequestSize uint64 = 512
+
 var (
 	// ErrNotFound is returned when there is no requested header.
 	ErrNotFound = errors.New("header: not found")
 
 	// ErrNoHead is returned when Store is empty (does not contain any known header).
 	ErrNoHead = fmt.Errorf("header/store: no chain head")
+
+	// ErrHeadersLimitExceeded is returned when ExchangeServer receives header request for more then RequestSize headers.
+	ErrHeadersLimitExceeded = fmt.Errorf("header/p2p: header limit per 1 request exceeded")
```

Comment:
Why are these errors `fmt.Errorf` if nothing is formatted? I think `errors.New` is what we use elsewhere